src/sass/layout/header.scss

```diff
@@ -0,0 +1,33 @@
+header {
+    height: 100vh;
```

Comment:
nie do konca jestem przekonany czy header powinien zajac 100% widoku mobile :). Wydaje mi sie ze nie, ale mozemy sprawdzic to pozniej jak dodasz kontent :)